proxy/proxy.go

```diff
@@ -358,23 +360,24 @@ func (p *proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	addBranding(rs)
 	metrics.MeasureBackend(rt.Id, start)
 
-	metrics.MeasureAllFiltersResponse(rt.Id, func() {
-		c.res = rs
-		if p.preserveOriginal {
-			c.originalResponse = cloneResponseMetadata(rs)
-		}
+	start = time.Now()
+	c.res = rs
+	if p.preserveOriginal {
+		c.originalResponse = cloneResponseMetadata(rs)
+	}
 
-		p.applyFiltersToResponse(f, c)
-	})
+	p.applyFiltersToResponse(f, c)
+	metrics.MeasureAllFiltersResponse(rt.Id, start)
 
 	if !c.Served() {
-		metrics.MeasureResponse(rs.StatusCode, r.Method, rt.Id, func() {
-			copyHeader(w.Header(), rs.Header)
-			w.WriteHeader(rs.StatusCode)
-			err := copyStream(w.(flusherWriter), rs.Body)
-			if err != nil {
-				log.Error(err)
-			}
-		})
+		start = time.Now()
+		copyHeader(w.Header(), rs.Header)
+		w.WriteHeader(rs.StatusCode)
+		err := copyStream(w.(flusherWriter), rs.Body)
+		if err != nil {
+			log.Error(err)
```

Comment:
previously, you were measuring errors as well. Now, you don't. Is that on purpose?